### PR TITLE
Conference page project

### PR DESCRIPTION
**Pull request by Willian-Berrocal**
- Setted up Linters
- Mobile version added
- Page deployed with GitHub Pages
- HTML files filled with content
- CSS styling done (for both mobile and desktop)
- Dynamic HTML done with JavaScript
- Menu buttons behavior done with JavaScript

video demo: https://www.loom.com/share/71dd87a93fcd47f6a2f22ae5a5dd18c0

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+const cardOccArr = document.querySelectorAll('.card-occupation');
+const cardDescArr = document.querySelectorAll('.card-desc');
+
+for (let i = 0; i < 6; i += 1) {
+  cardImgArr[i].src = speakersData[i].imgSrc;
+  cardImgArr[i].alt = speakersData[i].imgAlt;
+  cardNameArr[i].textContent = speakersData[i].name;
+  cardOccArr[i].textContent = speakersData[i].occupation;
+  cardDescArr[i].textContent = speakersData[i].desc;
+}
```
Comment (Abenezer-Tilahun) [inline main.js:66]: - [x] Good job for creating HTML elements with the data from each object and attach the new element to a parent inside within the loop, But I recommend you to use a function that iterates through the speakers data on window load .

`window.onload = () => {
  speakersData.map(speaker => {
    
  })
}
`

File: about.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      <div class="container-v">
+        <span>CC Global</span>
+        <span>Summit 2022</span>
+      </div>
+    </a>
+    <div>
+      <span>Copyright &copy; 2022</span>
+      <p>Some rights reserved.</p>
+    </div>
+  </footer>
```
Comment (Abenezer-Tilahun) [inline about.html]: - [ ] As stated in the project requirement You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors, Kindly adjust background color of the footer section just like the design guidelines and In given design there is no partner section on about page for desktop screen size , please fix it.

Your About page Look likes this : 
Capture

The Design guidelines About page look likes this : 
Capture

File: index.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+          <i class="fa-regular fa-image"></i>
+          <h3>Workshop</h3>
+          <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Repellat quia rerum deleniti.</p>
+        </div>
+        <div>
+          <i class="fa-solid fa-circle-nodes"></i>
+          <h3>Ignite</h3>
+          <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sequi enim eius tempore!</p>
+        </div>
+      </div>
```
Comment (Abenezer-Tilahun) [inline index.html:89]: [OPTIONAL] I kindly suggest you to add the white border on hover for all the program articles just like the design guidelines.

Your all the program articles design look likes this : 
Capture

The Design guidelines  program articles look likes this : 
Capture

File: index.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+          </div>
+        </div>
+        <div class="container-h">
+          <img class="card-img" src="" alt="">
+          <div>
+            <h3 class="card-name"></h3>
+            <span class="card-occupation"></span>
+            <p class="card-desc"></p>
+          </div>
+        </div>
```
Comment (Abenezer-Tilahun) [inline index.html:143]: - [ ] As stated in the project requirement the pages should look almost identical to the original design. So Kindly add transparent background image for all featured speakers just like the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and adjust background color for section.

The Design guidelines featured speakers section look likes this : 
Capture

Your  featured speakers section look likes this : 
Capture

[OPTIONAL] I kindly suggest  you  to add the line of text at the bottom to match the design, for mobile screen size.

The mobile screen size featured speakers section look likes this :

File: about.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+      </a>
+      <nav class="nav-bar">
+        <ul class="container-h">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+      </nav>
```
Comment (Abenezer-Tilahun) [inline about.html]: - [ ] It is redundant to have an `àbout` link on the about page. I kindly suggest you to replace this `about` page link with the `home` page link.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Willian-Berrocal  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional README 
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 6]
```diff
@@ -71,7 +71,7 @@ <h2 class="section-heading">See the past CC Global Summits</h2>
         <a href="#" class="summit-2020">CC Global Summit 2020</a>
       </div>
     </section>
```
Comment (mirouhml) [inline about.html]: - [x] Please know that as the previous reviewer mentioned you should follow the design guidelines, the images of the past summits should have a kind of orange overlay, you can use the main color to do so:
  image
  Kindly adjust this then submit again!

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Willian-Berrocal, 

Great work on making the changes requested by the previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, but don't worry you're almost there!! 😉

image

 ## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Willian-Berrocal , 
Congratulations!GIF

Well done on implementing the requested change. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**